tools/karpenter-convert/pkg/convert/convert.go

```diff
@@ -141,11 +160,21 @@ func (o *Context) RunConvert() error {
 	return nil
 }
 
-func Process(resource runtime.Object) (runtime.Object, error) {
+func parseOutput(buffer bytes.Buffer) string {
```

Comment:
```suggestion
func dropFields(buffer bytes.Buffer) string {
```

nit: naming that's more explicit?